pkg/controller/ibmblockcsi/ibmblockcsi_controller.go

```diff
@@ -493,6 +468,50 @@ func (r *ReconcileIBMBlockCSI) reconcileServiceAccount(instance *ibmblockcsi.IBM
 			if err != nil {
 				return err
 			}
+
+			controllerStatefulset, err := r.getControllerStatefulSetObject(instance)
+			if err != nil {
+				return err
+			}
+
+			nodeDaemonSet, err := r.getNodeDaemonSetObject(instance)
+			if err != nil {
+				return err
+			}
+
+			if strings.Contains(sa.Name, Controller) {
+				controllerlogger := log.WithValues("Resource Type", "Controller")
+				controllerPod := &corev1.Pod{}
+				err := r.getControllerPod(controllerStatefulset, controllerPod)
+				if err != nil {
+					controllerlogger.Error(err, "failed to get controller pod")
+					return err
+				}
+
+				controllerlogger.Info("controller requires restart",
+							"ReadyReplicas", controllerStatefulset.Status.ReadyReplicas,
+							"Replicas", controllerStatefulset.Status.Replicas)
+				controllerlogger.Info("restarting csi controller")
+				rErr := r.restartControllerPod(controllerPod)
+				
+				if rErr != nil {
+					return rErr
+				}
+			}
+			if strings.Contains(sa.Name, Node) {
+				nodelogger := log.WithValues("Resource Type", "Node DaemonSet")
+				nodelogger.Info("node rollout requires restart",
+				"DesiredNumberScheduled", nodeDaemonSet.Status.DesiredNumberScheduled,				
+				"NumberAvailable", nodeDaemonSet.Status.NumberAvailable)
```

Comment:
please indent function parameters wherever applicable, to align to current project convention

Reply:
do you have an example?

Reply:
e.g. [here](https://github.com/IBM/ibm-block-csi-operator/blob/9a30d70922dbfa46429c8ce6df9c16684510b76d/pkg/controller/ibmblockcsi/ibmblockcsi_controller.go#L420) (though it should be even a bit more indented, to be aligned with the 1st argument)

Reply:
done